term_test.go

```diff
@@ -1,42 +1,34 @@
-//+build linux
+// +build !windows
```

Comment:
oh; just noticed this doesn't seem to match the decription, which says this can now run both on Windows and Linux?

Reply:
It says it can run on multiple platforms.

Reply:
That is to say, it works on anything that's not windows... e.g. darwin.

Reply:
Ah, gotcha!